bin/git-checkout-plus

```diff
@@ -107,5 +107,9 @@ elif [[ "${1}" =~ '^https://github.com/([-_0-9a-zA-Z]+)/([-_0-9a-zA-Z]+)/pull/([
     fi
   fi
 else
-  git checkout "$@"
+  if git-branch-exists "$@"; then
+    git checkout "$@"
+  else
+    git checkout -b "$@"
```

Comment:
Perhaps a message "New branch created…" should also be output here to ensure user knows that branch did not previously exist.

Reply:
I often just run `gco -b foo`. I think this change would be fine as long as it only does it if there's only a single argument and it's not a flag (i.e. doesn't start with `-`).

Reply:
![image](https://user-images.githubusercontent.com/28444/84800877-46cbf580-afb3-11ea-892d-f22bcdfeee0a.png)